web/wp-content/themes/dctx2/template-flagexpress.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-z3j0iz.php, but found phpcs-scan-z3J0Iz.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).